.github/workflows/_docs.yml

```diff
@@ -53,6 +53,31 @@ on:
         description: Permissions for pushing to the docs site.
 
 jobs:
+  check-urls:
+    if: ${{ github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' }}
```

Comment:
@huydhn I naively hope it's gonna get triggered daily with this line, will it?

Reply:
Yeah, this only runs when the job is scheduled or dispatched manually.  As the name implies, it's a lint job, so should it run on pull_request like a normal doc build job?